doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,73 @@
+[[integrating_prometheus_alerts]]
+== OpenShift Prometheus Alerts
+
+[NOTE]
+====
+This feature is available with this release of {product-title} as a technology preview only. For more information on the support scope for features marked as technology previews, see https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope].
+====
+
+Prometheus is used as an external alerting component. {product-title} integrates Prometheus alerts for use with OpenShift Container Platform. {product-title_short} collects EmsEvents from Prometheus, generates alerts based on these events, and then attaches alerts to inventory objects.
+
+It is possible to view ongoing alerts in {product-title_short} and manage their life cycle, including the ability to:
+
+* View active alerts per provider in the alert dashboard.
+* View alert data and related objects.
+* Assign and acknowledge alerts in the alerts list screen.
+
+=== Integrating Prometheus Alerts in {product-title_short}
+
+Complete the following prerequisite steps on the OpenShift Container Platform:
+
+. Deploy and Configure Prometheus on OpenShift; see https://docs.openshift.com/container-platform/3.7/install_config/cluster_metrics.html#openshift-prometheus[Prometheus on OpenShift Container Platform].
+. Add Prometheus to an OpenShift cluster and configure alert definitions. 
+.. See intructions below to define alerts (currently has to be done on the Prometheus side):
++
+----
+$ oc edit configmap -n openshift-metrics prometheus
+# Supported annotations:
+# severity: ERROR|WARNING|INFO. defaults to ERROR.
+# miqTarget: ContainerNode|ExtManagementSystem, defaults to ContainerNode.
+# miqIgnore: "true|false", should ManageIQ pick up this alert, defaults to true.
+  prometheus.rules: |
+    groups:
+    - name: example-rules
+      interval: 30s # defaults to global interval
+      rules:
+      - alert: "Node Down"
+        expr: up{job="kubernetes-nodes"} == 0
+        annotations:
+          miqTarget: "ContainerNode"
+          severity: "ERROR"
+          url: "https://www.example.com/fixing_instructions"
+          message: "Node {{$labels.instance}} is down"
```

Comment:
We have a another kind of alert (`miqTarget: "ExtManagementSystem"`) that shows up differently so it's probably worth adding an example for that kind as well. To avoid confusion here is the entire block again:
```
# Supported annotations:
# severity: ERROR|WARNING|INFO. defaults to ERROR.
# miqTarget: ContainerNode|ExtManagementSystem, defaults to ContainerNode.
# miqIgnore: "true|false", should ManageIQ pick up this alert, defaults to true.
  prometheus.rules: |
    groups:
    - name: example-rules
      interval: 30s # defaults to global interval
      rules:
      # 
      # ------------- Copy below this line -------------
      #
      - alert: "NodeDown"
        expr: up{job="kubernetes-nodes"} == 0
        annotations:
          miqTarget: "ContainerNode"
          severity: "ERROR"
          url: "https://www.example.com/node_down_fixing_instructions"
          message: "Node {{$labels.instance}} is down"
      - alert: "TooManyRequests"
        expr: rate(authenticated_user_requests[2m]) > 12
        annotations:
          miqTarget: "ExtManagementSystem"
          severity: "ERROR"
          url: "https://www.example.com/too_many_requests_fixing_instructions"
          message: "Too many authenticated requests"
```

Reply:
Ack, updated.